server/chat-plugins/mafia.ts

```diff
@@ -698,8 +702,12 @@ class Mafia extends Rooms.RoomGame<MafiaPlayer> {
 		if (!(target in this.playerTable) && target !== 'novote') {
 			return this.sendUser(userid, `|error|${target} is not a valid player.`);
 		}
+		if (!this.votingall) return this.sendUser(userid, `|error|Voting is not allowed.`);
 		if (!this.enableNL && target === 'novote') return this.sendUser(userid, `|error|No Vote is not allowed.`);
 		if (target === player.id && !this.selfEnabled) return this.sendUser(userid, `|error|Self voting is not allowed.`);
+		if (this.votelock) {
+			if (player.voting) return this.sendUser(userid, `|error|You cannot switch your vote because votes are locked.`);
```

Comment:
```suggestion
		if (this.votelock && player.voting) {
			return this.sendUser(userid, `|error|You cannot switch your vote because votes are locked.`);
```

Reply:
fsr thought that was already in,ty